tests/roles/ceph_migrate/tasks/mon.yaml

```diff
@@ -103,6 +102,9 @@
     - ceph_drain
 
 - name: MON - Get current mon IP address
+  when:
+    - mon_ipaddr is not defined
+    - discover_mons | bool | default(true)
```

Comment:
discover_mons is experimental, right now the rdo job defines a static `mon_map` override to make sure we have predictable results.